localstack/services/awslambda/lambda_api.py

```diff
@@ -1076,6 +1093,108 @@ def delete_lambda_function(function_name: str) -> Dict[None, None]:
     return {}
 
 
+def get_lambda_url_config(api_id, region=None):
+    lambda_backend = LambdaRegion.get(region)
+    url_configs = lambda_backend.url_configs.values()
+    lambda_url_configs = [config for config in url_configs if config.get("CustomId") == api_id]
+    return lambda_url_configs[0]
+
+
+def event_for_lambda_url(api_id, path, data, headers, method) -> dict:
+    rawPath = path.split("?")[0]
+    rawQueryString = path.split("?")[1] if len(path.split("?")) > 1 else ""
+    queryStringParameters = (
+        {} if not rawQueryString else dict(urllib.parse.parse_qsl(rawQueryString))
+    )
+
+    requestContext = {
+        "acountId": "anonymous",
```

Comment:
We should know the account ID of the function owner (most likely 000000000000, but could be different). Can we also set this here?

Reply:
The documentation tells that the attribute should have the function's owner id, but it always returns anonymous at least when I run it against AWS.

I added the assertion in the test.